app/test_main.py

```diff
@@ -27,9 +27,9 @@
             id="Test invalid url"
         ),
         pytest.param(
-            False,
             True,
-            "Not accessible",
+            True,
+            "Accessible",
             id="Test valid url and have internet"
```

Comment:
This message should describe what you will receive if the test case fails. Pls fix that in all places